processes_notwindows.go

```diff
@@ -113,6 +113,7 @@ func readProcFile(filename string) ([]byte, error) {
 	data, err := ioutil.ReadFile(filename)
 	if err != nil {
 		if os.IsNotExist(err) {
+			// if file doesn't exists
 			return nil, nil
```

Comment:
Do we ignore the error in this case or do we have to return the `err`?

Reply:
Will clarify this in another PR

Reply:
hmmm not ideal but ok. I will open an issue an assign to you so we don't forget about this.